example/demo_app/pubspec.yaml

```diff
@@ -92,3 +92,4 @@ inno_bundle:
   license_file: assets/LICENSE.txt
   languages:
     - french
+  sign_tool: "signtool.exe sign /tr http://timestamp.digicert.com /td sha256 /fd sha256 /a $p $f"
```

Comment:
This added line results to the following when I try to run `dart run inno_bundle:build`

![image](https://github.com/user-attachments/assets/c2fe534e-6b6a-43fc-b82f-9ffb151395a8)